__tests__/index.js

```diff
@@ -5,19 +5,59 @@ import nock from 'nock'
 import {autocomplete, bulk, reverse, search} from '../index'
 
 const mockBulkResult = require('./mock-bulk-result.json')
+const mockBulkResultWithBadAddress = require('./mock-bulk-result-with-bad-address.json')
 const mockReverseResult = require('./mock-reverse-result.json')
 const mockSearchResult = require('./mock-search-result.json')
 const mockSuggestResult = require('./mock-suggest-result.json')
 
+function prepareNock (
```

Comment:
This is a partial rehash of a discussion we've had in the past. Simple functions scoped that take options arguments (`type`, `snapshotUri`, and `response`) should be avoided except in cases when it dramatically decreases the amount of code written. But that does not appear to be the case here. Composition of multiple functions or reusing variables would be much more ideal. Or use the Jest API.